earth_enterprise/src/common/fileaccessor_unittest.cpp

```diff
@@ -59,8 +62,52 @@ TEST(FileAccessorTest, validate_printf) {
     remove(filename.c_str());
 }
 
+TEST(FileAccessorTest, validate_getfileinfo) {
+    string filename = "/tmp/TestFile.txt";
+
+    time_t timeBeforeCreation = std::time(nullptr);
+
+    // Without a sleep here the difference between file creation time, modification
+    // time, and closing time would be too small to measure using time_t
+    auto sleepTime = std::chrono::seconds(1);
+    std::this_thread::sleep_for(sleepTime);
+
+    unique_ptr<AbstractFileAccessor> aFA = AbstractFileAccessor::getAccessor(filename);
+
+    const size_t fileSize = 200;
+    string theData = string(fileSize, 'X');
+
+    aFA->Open(filename, "w");
+    aFA->PwriteAll(theData.data(), fileSize, 0);
+    aFA->Close();
+
+    // Without a sleep here, the difference between file creation time, modification
+    // time, and closing time would be too small to measure
+    std::this_thread::sleep_for(sleepTime);
+
+    time_t timeAfterClose = std::time(nullptr);
+
+    uint64_t size = 0;
+    time_t mtime = 0;
+    
+    bool result = aFA->GetFileInfo(filename, size, mtime);
+    double creationToModification = std::difftime(mtime, timeBeforeCreation);
+    double modificationToClose = std::difftime(timeAfterClose, mtime);
+    std::cout << "\n\ncreationToModification = " << creationToModification <<
```

Comment:
Minor: this `cout` statement isn't really needed. I assume it was for debugging purposes. We could keep it or delete it - I'm fine either way.